src/components/TrackTile/hooks/useTrackers.ts

```diff
@@ -77,36 +73,34 @@ export const useTrackers = (trackersToUse?: Trackers) => {
     };
 
     const onRemoved = (tracker: Tracker) => {
-      if (isInstalledMetric(tracker)) {
-        if (tracker.system === TRACKER_PILLAR_CODE_SYSTEM) {
-          setPillarTrackers((currentTrackers) =>
-            sortTrackers(
-              currentTrackers?.map((t) => {
-                if (t.id === tracker.id) {
-                  return {
-                    ...omit(t, ['target', 'unit', 'metricId']),
-                    id: tracker.metricId,
-                  };
-                }
-                return t;
-              }),
-            ),
-          );
-        } else {
-          setTrackers((currentTrackers) =>
-            sortTrackers(
-              currentTrackers?.map((t) => {
-                if (t.id === tracker.id) {
-                  return {
-                    ...omit(t, ['target', 'unit', 'metricId']),
-                    id: tracker.metricId,
-                  };
-                }
-                return t;
-              }),
-            ),
-          );
-        }
+      if (tracker.system === TRACKER_PILLAR_CODE_SYSTEM) {
```

Comment:
It'll definitely be good to heavily test both trackers and pillars after these changes.

Reply:
Yeah great point. I think we are fairly insulated here since we don't have the ability to remove a pillar at the moment. This change makes it so we run the logic every time and set `installed` to `false`.

I will definitely play around with pillars before merging just to make sure everything is okay.